hip/settings/base.py

```diff
@@ -198,6 +198,8 @@
 AWS_ACCESS_KEY_ID = os.getenv("AWS_ACCESS_KEY_ID")
 AWS_SECRET_ACCESS_KEY = os.getenv("AWS_SECRET_ACCESS_KEY")
 
+# Messages
+MESSAGE_STORAGE = "django.contrib.messages.storage.session.SessionStorage"
```

Comment:
We shouldn't need this right? If we don't set it, the messages framework will use cookies first and then fallback to sessions if necessary: https://docs.djangoproject.com/en/3.1/ref/contrib/messages/#storage-backends

Reply:
done